integration/spark/src/main/scala/org/apache/spark/sql/parser/CarbonSparkSqlParserUtil.scala

```diff
@@ -683,12 +683,6 @@ object CarbonSparkSqlParserUtil {
       fields: List[Field],
       tblProp: Option[List[(String, String)]]
   ): CarbonAlterTableAddColumnCommand = {
-    fields.foreach { f =>
```

Comment:
i think since you are supporting only for single level complex column, if user tries to add multi level you should throw exception, and wherever you handle this, please add a TODO and mention jira ID which you have already created to support multilevel

Reply:
done